Makefile

```diff
@@ -30,16 +34,20 @@ IMAGE := ${DEIS_REGISTRY}/deis/${SHORT_NAME}:${VERSION}
 all:
 	@echo "Use a Makefile to control top-level building of the project."
 
+# Containerized dependency resolution / initial workspace setup
+bootstrap: check-docker
+	${DEV_ENV_CMD} glide up
+
 # This illustrates a two-stage Docker build. docker-compile runs inside of
 # the Docker environment. Other alternatives are cross-compiling, doing
 # the build as a `docker build`.
 build: check-docker
 	mkdir -p ${BINDIR}/bin
-	docker run --rm -v ${PWD}:/app -w /app golang:1.5.1 make docker-compile
+	${DEV_ENV_CMD} make docker-compile
 
 # For cases where build is run inside of a container.
 docker-compile:
```

Comment:
fair point. I'm just thinking that giving the choice could add some confusion to the build process.

also, I don't think it's a big deal to have to download a ~25mb image once and leave the VM running forever